Лабораторная работа 1

Amizx2 opened the pull request:
![изображение](https://github.com/user-attachments/assets/d77328fc-771f-4e27-8c98-32e60fd6ac62)

Amizx2 committed:
Add files via upload

ZebraHead22 reviewed (commented):
1.  А если None в другом месте?..Код придется менять. Надо сразу присать универсальный код...

Amizx2 commented:
> 1. А если None в другом месте?..Код придется менять. Надо сразу присать универсальный код...

Здравствуйте! Это верно, но если делать без циклов и условий (например) и без индексов (.index) то я не придумал другого решения. Могу сделать с помощью .index

Amizx2 committed:
FIx task_1.py

Добавил поиск элемента "None" через метод .index, которая возвращает индекс необходимого элемента.
